pkg/addons/data/ConfigMap/openshift-etcd/backup-config.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v1
```

Comment:
Do we need to keep `pkg/addons/data/EtcdBackup.etcd.database.coreos.com/openshift-etcd/etcd-backup.yaml` with this?

Reply:
Nevermind, I see you remove this in this PR.